docs/USAGE.md

```diff
@@ -0,0 +1,296 @@
+
+# Contents
+
+- [Installation Notes](#installation-notes)
+- [Using Operations and Block Paths Properly](#using-operations-and-block-paths-properly)
+  - [Block Path](#block-path)
+  - [Rename and Remove Operations](#rename-and-remove-operations)
+  - [Alter Block Structure Operations](#alter-block-structure-operations)
+  - [Other Operations](#other-operations)
+- [Making Custom Operations](#making-custom-operations)
+  - [Basic Usage](#basic-usage)
+  - [block_value](#blockvalue)
+  - [Making Structural Changes](#making-structural-changes)
+  - [Old List Format](#old-list-format)
+
+# Installation Notes
+
+- **For now we only support Wagtail 4**
+
+- Note that the migration operation `MigrateStreamData` is dependent on the migrations of wagtailcore.
```

Comment:
I think you could do with an example of basic usage before you start diving into exact definitions of block path etc. It's not very clear that you need to make a streamfield change, create a new migration file with a new `MigrateStreamData` operation before you move on to the next step. Also, we're not dependent on all wagtailcore migrations - maybe worth just noting which one we need!